packages/quick_blue/lib/src/quick_blue_platform_interface.dart

```diff
@@ -45,6 +45,8 @@ abstract class QuickBluePlatform extends PlatformInterface {
 
   Stream<dynamic> get scanResultStream;
 
+  Stream<dynamic> get availabilityChangeStream;
```

Comment:
Move `get availabilityChangeStream` above `startScan` and below `isBluetoothAvailable`